views/index.erb

```diff
@@ -54,6 +85,11 @@
 
   // TODO - Count the number of words in the profile.
   // hint - look at what the innerHTML property provides.
+  for(var i = 0; i < profile_values.length; i++) {
+    //profile_values are HTML Elements. Get value.
+    var text = profile_values[i].innerHTML;
+    total_words += text.split(' ').length;
+  }
 
   document.getElementById('word-count').innerHTML = total_words;
 </script>
```

Comment:
This file is missing a terminating newline. Depending on your editor of choice you can usually auto-configure one to be added upon saving a file. Take a look here to understand why this is important: http://stackoverflow.com/a/729795